csharp-basic-topics/AwaitInForEachLoopCsharp/AwaitInForEachLoopCsharp/AwaitInForEachLoop.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+namespace AwaitInForEachLoop
+{
+    public static class AwaitInLoop
+    {
+        public static void PrepNumbers()
+        {
+            Console.WriteLine("Incoming number");
+        }
+        public static async Task<List<int>> ResultAsync()
+        {
+            List<int> numbers = [10, 20, 30, 40, 50, 60, 70, 80, 90];
+
+            foreach (var number in numbers)
+            {
+                PrepNumbers();
```

Comment:
Why we need to call this static method and print the same value inside the loop?

Reply:
Should I take the prepnumbers() method out of the loop? It's just a console.writeline.

Reply:
You can remove the method. If you want you can use Console.WriteLine before the loop and print the message something as "Start Processing"